geniza/corpus/templates/admin/corpus/document/change_form.html

```diff
@@ -0,0 +1,15 @@
+{% extends "admin/change_form.html" %}
+{% block after_field_sets %}
+<fieldset class="module aligned">
+    <div class="form-row">
+        <div class="fieldBox">
+            <label>First input:</label>
```

Comment:
Earliest revision, maybe?

Reply:
Can it even be a revision if it's the first one?

Reply:
😆 🤔 too early to be asking existential questions like that!

first version? initial entry?